### DISCUSSION TITLE
Moving TorchIO to https://github.com/TorchIO-Project

### DISCUSSION THREAD

**Discussion by fepegar** [TorchIO-project/torchio#1067] [General]
Hello TorchIO community!

As the creator of the TorchIO, I'm incredibly grateful for the support and contributions from each of you. Your support has been instrumental in making TorchIO a success. The project has now gained significant momentum:

- The paper has [255 citations, according to Google Scholar](https://scholar.google.com/citations?view_op=view_citation&hl=en&user=Gc2eg3kAAAAJ&citation_for_view=Gc2eg3kAAAAJ:WF5omc3nYNoC)
- [~1700 stars on GitHub](https://github.com/fepegar/torchio/stargazers)
- [49 contributors on GitHub](https://github.com/fepegar/torchio/graphs/contributors), or [more, according to allcontributors](https://github.com/fepegar/torchio#contributors)
- [~18,000 downloads/month](https://pypistats.org/packages/torchio)

Due to personal and professional commitments, I find myself with limited time to maintain the project to the best of my abilities. In order to ensure the continued maintenance and growth of TorchIO, I propose moving the repository to [https://github.com/TorchIO-Project](https://github.com/TorchIO-Project) and inviting some of our most active contributors to join as moderators.

Moving the repository will allow for a more decentralized and collaborative approach to maintaining and developing the library while retaining the values and goals we have held since the project's inception.

I would like to invite the following contributors as moderators, based on their consistent and significant contributions to the project:

- @romainVala
- @justusschock
- @dmus 
- @jmlipman
- @GFabien
- @JulianKlug
- @dvolgyes
- @efirdc
- @ramonemiliani93
- @iimog

If you have been mentioned above, please let me know below if you'd rather not be invited (feel free to email me if you prefer). Your commitment to the project has not gone unnoticed, and I truly believe that your expertise will be invaluable in guiding TorchIO's future.

If anyone else in the community is interested in getting more involved in maintaining the project or has suggestions for other potential moderators, please feel free to comment below.

Lastly, I want to emphasize that I am not leaving the project! I will still be involved and available for guidance, albeit in a more limited capacity compared to the first couple of years.

Thank you all once again for your amazing support and contributions to TorchIO. I'm excited to see how the project will evolve under the guidance of our new moderators and the continued input of the community.

**Comment by romainVala** [comment]
Hi fernando

thanks you for this nice work. It is, indeed, a significant contribution to the field. (as shown by this success)
I would be happy if I could help to continue this project

I just wonder in a longer perspective if MONAI will not replace torchio. They have more and more transform that re-implement what is done in torchio. I did not test them carefully, but I think they still miss some essential part like the patch based sampling with the queue functionalities. There may still be today some unique transform (like motion) in torchio, but it should not be that much work to reimplement them in MONAI ... no ?

what do you (and other)  think on this ?

**Comment by dmus** [comment]
Very nice project. I would be happy to help. We build our framework on top
of torchio because of its simplicity and it feels intuitive to use. I agree
that MONAI is a framework with lots of possibilities and maybe it will
replace torchio, but it can be a bit overwhelming.

Op di 11 apr 2023 om 08:23 schreef valabregue ***@***.***>:

**Comment by ramonemiliani93** [comment]
Count me in! I'm glad to help with TorchIO.

**Comment by fepegar** [comment]
Thank you all ✨

MONAI is a great project, with a very large scope. It is always growing and has a lot of resources (NVIDIA, Kitware, King's College London...). TorchIO is little, but focused, and tries to provide a nice user experience, which is also beginner-friendly. I think they're related but slightly different projects. I'm not sure how much effort it would take to reimplement things into MONAI, nor why that would be needed. I certainly couldn't really afford to do that!

**Comment by dvolgyes** [comment]
@fepegar Thanks for remembering me. 😀
First and foremost: I think moving to community maintenance is a good idea. 

However, I think I am in a similar situation as you: I try to stay around the community,
but I have very limited time, and I actually moved to remote sensing as primary field of interest.
(Quite similar issues. And damn clouds. But if anyone wants to change their field, 
medical experience is quite helpful in remote sensing.)

So I am afraid I can't take moderator responsibility, but I will try to contribute here and there when I can.

**Comment by jmlipman** [comment]
@fepegar thanks for the invitation. TorchIO **really** helped me, ultimately, saving me a lot of time. It was an honor and a pleasure to give back a little of the time that TorchIO saved me in form of a small contribution to the project.

Similarly to @dvolgyes, I moved to a slightly different field; in my case, the images I'm going to be working on are at the border of "medical imaging". I will also be quite busy due to family reasons but I will try to stay around : )

To date, TorchIO has MRI-specific transformations that cannot be found in MONAI so it has a lot of value and potential.

**Comment by fepegar** [comment]
Hi all. I would like this to happen soon. @nicoloesch, would you like to have maintainer rights in the repository?

**Reply by nicoloesch** [reply]
@fepegar Yes, I’d be happy to take on maintainer rights. I appreciate the opportunity and will do my best to contribute effectively. Let me know if there are any specific guidelines or responsibilities I should be aware of
